### DISCUSSION TITLE
Does portable_pty expose stderr separate from stdout?

### DISCUSSION THREAD

**Discussion by roberte777** [wezterm/wezterm#5529] [Q&A]
As part of the Rust Command API, I can get stderr and stdout separately. Am I able to do something like this with portable_pty? As far as I can tell, I can only get one reader which gets both outputs (I may be incorrect about this as well). 

Any help appreciated, thanks!

**Comment by wez** [comment]
See: https://github.com/wez/wezterm/discussions/3948#discussioncomment-6391906

**Reply by roberte777** [reply]
Perfect, thank you!! I was having trouble finding anything related. :D
